src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
should this use the `@expanding-table-line-height` variable instead of 25px hardcoded?